### PR TITLE
Feat/message/mempool

### LINKED ISSUE #82: mempool

### PR DESCRIPTION
**Pull request by oxlime**
closes #82 
Added mempool.zig, implementing necessary functions to deserialize/serialize into the wire
Added some tests but should look similiar to Verack as neither contain payload.

### CODE REVIEW

File: src/network/protocol/messages/mempool.zig [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
+const std = @import("std");
+const native_endian = @import("builtin").target.cpu.arch.endian();
+const protocol = @import("../lib.zig");
+
+/// MempoolMessage represents the "mempool" message
+///
+/// https://developer.bitcoin.org/reference/p2p_networking.html#version
```
Comment (tdelabro) [inline src/network/protocol/messages/mempool.zig:8]: wrong link

**Review by tdelabro — CHANGES_REQUESTED** [review]
You should also add it as an if/else condition in wire deserializeMessage

**Comment by oxlime** [comment]
> You should also add it as an if/else condition in wire deserializeMessage

should be good now i just forgot to git add that file.

**Comment by oxlime** [comment]
> You should also add it as an if/else condition in wire deserializeMessage

https://github.com/oxlime/btczee/blob/cae11b088a2b052dc768b84612eb503f81b52005/src/network/wire/lib.zig#L83

**Comment by tdelabro** [comment]
oh I missed it sorry

**Review by tdelabro — APPROVED** [review]

**Pull request merged**